### DISCUSSION TITLE
COOKIES_FILE is not working with instagram

### DISCUSSION THREAD

**Discussion by patrickclery** [ArchiveBox/ArchiveBox#861] [Q&A]
Has anyone been able to successfully archive instagram pages using `COOKIES_FILE`? (Or any other methods?)

When using `COOKIES_FILE`, I still see the "Login" page.

Perhaps my setup is broken - can anyone confirm they've been able to snapshot an instagram page?

Thanks :)

**Comment by pirate** [comment]
COOKIES_FILE only applies to the wget method, you need CHROME_USER_DATA_DIR and COOKIES_FILE set to have most of the methods work.
